src/ray/gcs/format/gcs.fbs

```diff
@@ -337,8 +337,9 @@ table ActorCheckpointData {
 table ActorCheckpointIdData {
   // ID of this actor.
   actor_id: string;
-  // A list of the available checkpoint IDs for this actor.
-  checkpoint_ids: [string];
+  // IDs of this actor's available checkpoints.
+  // Note, this is a long string that concatenates all the IDs.
+  checkpoint_ids: string;
```

Comment:
Why the change from `[string]` to `string`? This seems more confusing...

Reply:
it's because flatbuffer generated Java class cannot get a single id from the list. We've already changed other `[string]` to long strings. See https://github.com/ray-project/ray/pull/3749